metrics/codahale_test.go

```diff
@@ -516,3 +516,26 @@ func TestCodaHaleServeMetrics(t *testing.T) {
 		})
 	}
 }
+
+func BenchmarkMeasureSince(b *testing.B) {
+	c := NewCodaHale(Options{})
+	now := time.Now()
+
+	b.Run("measureSince", func(b *testing.B) {
```

Comment:
Why do we need to use sub benchmark here instead of just having the loop directly?

Reply:
I can drop it if you like but I don't see that this is bad.

Reply:
It is not wrong, just not clear why